Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-data-processing.controller.ts

```diff
@@ -0,0 +1,17 @@
+﻿((ng, app) => {
+    app.config(["$stateProvider", $stateProvider => {
+        $stateProvider.state("it-system.usage.dataprocessing", {
+            url: "/dataprocessing",
+            templateUrl: "app/components/it-system/usage/tabs/it-system-usage-tab-data-processing.view.html",
+            controller: "systemUsage.DataProcessing"
+        });
+    }]);
+
+    app.controller("systemUsage.DataProcessing", ["$scope",
```

Comment:
Please use the TS controller style where you actually add a class. This is the old school style